### ISSUE TITLE
Multiple views in customizable layout?

### ISSUE THREAD

**Issue by careylam** [mozilla/side-view#310]
Support multiple side-views on the left, right, bottom, and/or top. 
[Untitled document.pdf](https://github.com/mozilla/side-view/files/2249015/Untitled.document.pdf)

**Comment by ianb** [comment]
Screenshot of the attached PDF: 
<img width="523" alt="image" src="https://user-images.githubusercontent.com/44390/43533531-a1408a12-957a-11e8-93e5-bb4124de2a8b.png">

**Comment by careylam** [comment]
Yeah, it's somewhat similar to Chrome's experimental feature, "Web Panels". It has been removed due to the maintenance cost.
